README.md

```diff
@@ -20,6 +20,14 @@ If you need more information, check out our [VIP CLI documentation](https://vip.
 
 ## Changelog
 
+### 1.2.0
+
+- We now display a preview of the `sync` feature with the backup time and the search/replace taking place in your database.
+- We now display your mapped domain instead of the `vip-go.co` ones in `vip app` and `vip app list`.
+- Bug fixes.
```

Comment:
It would be nice to mention the specific fixes (if we can).

Reply:
yes, i'll put this on hold until we finish everything for the needed for the `v1.2`

Reply:
changed it with the help fix here #181